content/docs/quickstart/cpp.md

```diff
@@ -127,32 +219,34 @@ Remember to save the file!
 
 ### Generate gRPC code
```

Comment:
nit: "Regenerate gRPC code" may be better because we're generating again after we've updated the .proto file?

Reply:
Done